### DISCUSSION TITLE
Add Support for SolidJS

### DISCUSSION THREAD

**Discussion by thedanchez** [emotion-js/emotion#2981] [Ideas]
Hello, wanted to ask if there are any plans to add support for SolidJS in Emotion? Would love to see a `@emotion/solid` package like there is for `@emotion/react`. SolidJS is pretty similar to React in many ways and having a package like `@emotion/solid` would help people who are porting their React apps over to Solid make the transition easily.

**Comment by Andarist** [comment]
I don't have the capacity to work on this but I could potentially review the PR implementing such a package.
